### PR TITLE
fix: remove all ES6 syntax

### PR DESCRIPTION
**Pull request by malwilley**
I have the unfortunate need to support IE11, so I noticed some places that were using non-ES5 features. 

I've tested on v0.3.1 and can confirm that IE11 works with these changes (and the polyfill provided in the readme).

### CODE REVIEW

File: src/runtime/refreshUtils.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ function getReactRefreshBoundarySignature(moduleExports) {
     return signature;
   }
 
-  for (const key in moduleExports) {
+  for (let key in moduleExports) {
```
Comment (pmmmwh) [inline src/runtime/refreshUtils.js:30]: I think there's no need to switch to `let` from `const` since IE11 supports both?

Comment (malwilley) [reply src/runtime/refreshUtils.js:30]: That was my thought as well! However, it appears that IE11 has one small caveat: https://caniuse.com/#feat=const

> Not supported in for-in and for-of loops

😞

Comment (pmmmwh) [reply src/runtime/refreshUtils.js:30]: OH MY GOD IE WHY 🤦

**Comment by malwilley** [comment]
Thanks for the swift response!

**Comment by pmmmwh** [comment]
`v0.3.2` published.

**Pull request merged**